packages/runtime/container-runtime/src/test/garbageCollection.spec.ts

```diff
@@ -131,6 +131,9 @@ describe("Garbage Collection Tests", () => {
 				return { totalNodeCount: 0, unusedNodeCount: 0 };
 			},
 			updateUnusedRoutes: (unusedRoutes: string[]) => {},
+			deleteUnusedNodes: (deletableRoutes: string[]): string[] => {
+				return [];
```

Comment:
We don't have any tests that inspect the return value here?

Reply:
This is an implementation of the `IGarbageCollectionRuntime`, so technically it can return anything. Furthermore, the result of this will further change with the subsequent blob sweep